api/anomaly/services/plotly.py

```diff
@@ -0,0 +1,82 @@
+
+from anomaly.models import Anomaly
+import pandas as pd
+import plotly.graph_objects as go
+
+class Plotly:
+
+    @staticmethod
+    def anomalyChartToImgStr(anomalyId: int = 0):
+        """ Generate anomaly chart and convert it to image """
+        
+        anomaly = Anomaly.objects.get(id=anomalyId)
+        granularity = anomaly.anomalyDefinition.dataset.granularity
+        data = anomaly.data
+        anomalyData = data["anomalyData"]
+        actualData = anomalyData["actual"]
+        predictedData = anomalyData["predicted"]
+        bandData = anomalyData["band"]
+        actualDataDf = pd.DataFrame(actualData)
+        predictedDataDf = pd.DataFrame(predictedData)
+        band_lower_y = [ band["y"][0] for band in bandData]
+        band_upper_y = [ band["y"][1] for band in bandData]
+        band_x = [ band["ds"] for band in bandData]
+        anomalyDataDf = actualDataDf.loc[actualDataDf['anomaly'] == 15]
+        anomaly_x = anomalyDataDf["ds"].to_list()
+        anomaly_y = anomalyDataDf["y"].to_list()
+        actual_x = actualDataDf["ds"].to_list()
+        actual_y = actualDataDf["y"].to_list()
+        predicted_x = predictedDataDf["ds"].to_list()
+        predicted_y = predictedDataDf["y"].to_list()
```

Comment:
band_lower_y
band_upper_y
band_x
anomaly_x
anomaly_y
actual_x
actual_y
predicted_x
predicted_y


please change the variable names to camelCase convention.